.yamllint.yml

```diff
@@ -0,0 +1,18 @@
+extends: default
+
+ignore-from-file: .gitignore
+
+rules:
+  document-start:
+    present: false
+
+  # 80 chars should be enough, but don't fail if a line is longer
+  line-length:
+    max: 80
+    level: warning
+    allow-non-breakable-words: true
+    allow-non-breakable-inline-mappings: true
+
+  # don't bother me with this rule
+  indentation: disable
```

Comment:
I think consistency on indentation would be great (one of the things I like best about linting!). What do you think about:

```
rules:
  indentation:
    spaces: 2
    indent-sequences: consistent
    check-multi-line-strings: false
```